dts/bindings/clock/st,stm32-rcc.yaml

```diff
@@ -58,12 +58,22 @@ properties:
       required: false
       enum:
         - 1
+        - 2
+        - 4
+        - 8
+        - 16
+        - 32
```

Comment:
32 is not a valid value , at least for STM32G4 (to be checked for other series)

Reply:
correct except for stm32wb/wl series. I think it is better to allow any value in the range (DTS) and check which one is accpeted depending on the soc serie, Like what is done in  the _clock_stm32_ll_common.c_ for the  `fn_apb2_prescaler` of the  `CONFIG_SOC_SERIES_STM32F0X`